bots/images/scripts/candlepin.setup

```diff
@@ -14,24 +14,37 @@ openssl \
 
 yum -y install $CANDLEPIN_DEPS
 
-# Check out candlepin; HACK: newer releases split out Ansible roles to
-# https://github.com/candlepin/ansible-role-candlepin, but these playbooks are
-# completely broken with Ansible 2.6.
-git clone --depth=1 -b candlepin-2.5.1-1 https://github.com/candlepin/candlepin.git /root/candlepin
+mkdir -p playbookdir; cd playbookdir;
 
-# Insert our variables and change parameters
-cp /var/lib/testvm/candlepin-defaults.yml /root/candlepin/vagrant/roles/candlepin-user/defaults/main.yml
-
-# The playbook expects buildr to be linked from the home dir for some reason
-mkdir /root/bin
-ln -sf /usr/local/bin/buildr /root/bin/buildr
+mkdir -p roles
+git clone https://github.com/candlepin/ansible-role-candlepin.git roles/candlepin
 
 # Run the playbook
-cat > /root/inventory <<- EOF
+cat > inventory <<- EOF
 [dev]
 localhost
 EOF
-ansible-playbook -i /root/inventory -c local -v /root/candlepin/vagrant/candlepin.yml
+
+useradd -m admin
```

Comment:
I don't quite understand how *just* this command makes `admin` able to run sudo. It would at least need to be in the `wheel` group. But as apparently the deployment script needs passwordless sudo, so let's  do that:
```
echo admin:foobar | chpasswd
echo 'admin ALL=(ALL) NOPASSWD: ALL' > /etc/sudoers.d/admin
```

Reply:
I think it expects passwd admin for admin user, https://github.com/cockpit-project/cockpit/blob/master/test/verify/check-subscriptions#L32

Reply:
Outrageous! The only valid password is "foobar"!!! :stuck_out_tongue_winking_eye: 

This comment seems long obsolete, obviously the previous image didn't even have a non-root user configured. But I don't feel like force-pushing yet again for this.